src/backstop/backstop_user_types.ts

```diff
@@ -73,14 +87,16 @@ export class UserBalance {
 
     let shares: bigint | undefined;
     let q4w: Q4W[] = [];
+    let unlockedQ4W: Q4W[] = [];
```

Comment:
this should just be a bigint

The protocol does not treat unlocked Q4W entries differently, so it is essentially is the same.